src/examples/signTaskDueDate.js

```diff
@@ -0,0 +1,61 @@
+// An example using the setTaskDueDate operation
+const signTaskDueDate = async (colonyClient, taskId) => {
+
+  // Get JSON formatted operation from the mock database
+  const operationJSON = DATABASE.setTaskDueDateOperationJSON;
+
+  // Check the operation
+  if (operationJSON) {
+
+    // Restore operation
+    const operation = await colonyClient.setTaskDueDate.restoreOperation(
+      operationJSON,
+    );
+
+    // Check if the operation matches the colony contract and the task id and
+    // check if current account is required to sign the operation.
+    if (
```

Comment:
Should we perhaps log something out if this check doesn't pass?

Reply:
:+1: